tests/test_history.py

```diff
@@ -241,7 +245,12 @@ def create_fake_minute_data(cls, tempdir):
                                    "DIVIDEND_minute.csv.gz"),
         }
 
-        MinuteBarWriterFromCSVs(resources).write(tempdir.path, cls.assets)
+        path = os.path.join(tempdir.path, 'equity', 'minutes')
```

Comment:
worth calling this `equities_tempdir` to be consistent with `futures_tempdir` above?